scripts/migration-task-script.py

```diff
@@ -1,36 +1,100 @@
 #!/usr/bin/env python3
+import re
 import subprocess
 import sys
 
-command_to_run = "flask db upgrade"
+
+def get_db_revision_id() -> str | None:
+    try:
+        grep_result = subprocess.check_output(
+            (
+                "copilot svc exec --name post-award "
+                "--command \"bash -c 'launcher flask db current 2> /dev/null | grep head'\""
+            ),
+            shell=True,
+            text=True,
+        )
```

Comment:
I think a short comment here explaining why we're doing all this malarky wouldn't hurt - i.e. we need the output of the copilot svc exec command to be short because copilot svc exec uses AWS ssm session-manager, which isn't designed to run in github actions (where it doesn't have a TTY). Too much output leads to output buffering, which leads to github actions not getting all of the output (specifically the line of output we need), leading to the script failing.

So we're doing a bit of a song-and-dance to make the command ran inside copilot svc exec only generate a few lines of output, which is short enough for github actions to reliably read, which means we can get the DB revision ID